conda_smithy/configure_feedstock.py

```diff
@@ -1008,7 +1008,7 @@ def _get_platforms_of_provider(provider, forge_config):
         if provider in providers:
             platforms.append(platform)
             archs.append(arch)
-            if platform == "linux" and arch == "64":
+            if platform_arch in forge_config["noarch_platform"]:
```

Comment:
I think you need this line on the if statement just below here too.

Reply:
nice db3ce03